docs/CHANGELOG.md

```diff
@@ -9,15 +9,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 The format of this change log follows the advice given at [Keep a CHANGELOG](http://keepachangelog.com).
 
 ## [Unreleased]
-## Changed
+### Changed
+- TODO: Document all the changes to jump to this new release from previous 2.5.0, similarly to the docs explaining 1.x => 2.x changes.
+- ACTION REQUIRED: project organization renamed to moodlehq. You must update your `.travis.yml` to use `moodlehq/moodle-plugin-ci`
 - Updated [.travis.dist.yml] with a new `services` section to ensure databases start.
-- Updated [.travis.dist.yml] to remove `openjdk-8-jre-headless`. See
-  [#91](https://github.com/blackboard-open-source/moodle-plugin-ci/issues/91).
-- Updated verion of `moodlehq/moodle-local_ci` to fix Mustache linting. See
-  [#91](https://github.com/blackboard-open-source/moodle-plugin-ci/issues/91).
+- Updated [.travis.dist.yml] to remove `openjdk-8-jre-headless` and updated `moodlehq/moodle-local_ci` to fix Mustache linting. See [moodle-local_ci/pull#198](https://github.com/moodlehq/moodle-local_ci/pull/1989).
 - Replaced Selenium in-built functionality with docker image for Selenium
-  Standalone server. See [#99](https://github.com/blackboard-open-source/moodle-plugin-ci/issues/99).
-- Updated version of `moodlehq/moodle-local_codechecker` to 2.9.6
+  Standalone server. See [#5](https://github.com/moodlehq/moodle-plugin-ci/pull/5).
```

Comment:
In that PR #5 there was a fix to standalone server (Firefox version), it was actually implemented in https://github.com/blackboard-open-source/moodle-plugin-ci/pull/107

Reply:
Ah, I think I ended confusing things because the original link was to 99, not 107. Now I see it. Fix ongoing...